modules/sample-dropwizard/src/test/scala/core/Dropwizard/JavaInvalidCharacterEscapingTest.scala

```diff
@@ -0,0 +1,10 @@
+package core.Dropwizard
+
+import org.scalatest.freespec.AnyFreeSpec
+import org.scalatest.matchers.must.Matchers
+
+class JavaInvalidCharacterEscapingTest extends AnyFreeSpec with Matchers {
+  "Invalid characters should be escaped to reasonable things" in {
+    
```

Comment:
the best kind of tests are the ones that can't be written, well done.

Reply:
Lol oops.  Too much multitasking.